src/main/java/com/chaosthedude/naturescompass/util/BiomeSearchWorker.java

```diff
@@ -104,10 +104,10 @@ public boolean doWork() {
 	private void finish(boolean found) {
 		if (found) {
 			NaturesCompass.logger.info("Search succeeded: " + getRadius() + " radius, " + samples + " samples");
-			((ItemNaturesCompass) stack.getItem()).setFound(stack, x, z, samples, player);
+			((ItemNaturesCompass) stack.getItem()).setFound(stack, x, z, samples, maxDistance, player);
 		} else {
 			NaturesCompass.logger.info("Search failed: " + getRadius() + " radius, " + samples + " samples");
-			((ItemNaturesCompass) stack.getItem()).setNotFound(stack, player, getRadius(), samples);
+			((ItemNaturesCompass) stack.getItem()).setNotFound(stack, player, maxDistance, samples);
```

Comment:
Rather than reporting the actual distance searched (since you could use that to triangulate) we just parrot back the requested radius.